plugins/org.eclipse.che.editor.gwt/1.0.0/meta.yaml

```diff
@@ -6,3 +6,12 @@ title: Eclipse GWT IDE for Eclipse Che
 description: Eclipse GWT IDE
 icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
 url: https://github.com/eclipse/che-editor-gwt-ide/releases/download/che-editor-gwt-ide-2018-12-18_2212-00/che-editor-plugin.tar.gz
+publisher: Red Hat, Inc.
+category: Editor
+repository: https://github.com/eclipse/che-editor-gwt-ide/
+firstPublicationDate: Jan 11, 2019
+latestUpdateDate: Jan 11, 2019
```

Comment:
this field could be also automatically generated when building the image

Reply:
I mean, the field is important when we read data not when we create yaml file.

AFAIK the script could add the fields when building the image if missing